Introduction/tutorials/Introduction.md

```diff
@@ -1,106 +1,404 @@
+
+:question: Question: How many exons are there in total in your file? (*Hint: each line now represents a different exon, so you can see the answer to this when you expand the history item, as in the image above*)
+
+## Sorting exons by SNPs count
+
+Now we have a list of all exons and the number of SNPs they contain, but we would like to know which exons has the *highest number* of SNPs, we can do this by sorting the file on the second column.
 
 :pencil2: ***Hands on!***
 
-1. First step
-2. Second step
-3. Third step
+1. Navigate to the tool `Sort - data in ascending or descending order`
+
+2. Set the **on column** parameter to `Column: 2`, by default it will select a numerical sort in descending order, which is exactly what we want in this case.
+
+ ![](../images/101_15.png)
+
+3. Click **Execute** and examine the output file.
+
+ ![](../images/101_16.png)
+
+You should now see the same file as we had before, but the exons with the highest number of SNPs are now on top.
 
-## Subpart 2
+:question: Question: Which exon has the highest number of SNPs in your file (remember, it is okay if this is different to the screenshot above)
 
-Short introduction about this subpart.
+## Selecting top five exons
+
+Let's say we want a list with just the top-5 exons with highest number of SNPs.
 
 :pencil2: ***Hands on!***
 
-1. First step
-2. Second step
-3. Third step
+1. Open the tool `Select first - lines from a dataset`
+
+2. Set **select first** to `5` and choose the sorted dataset from the previous step as the input.
+
+ ![](../images/101_17.png)
+
+3. Click **Execute** and examine the output file, this should contain only the first 5 lines of the previous dataset.
+
+## Recovering exon info and displaying data in genome browsers
 
-Some blabla
+Congratulations! you have now determined which exons on chromosome 22 have the highest number of SNPS, but what else can we learn about them? One way to learn more about a genetic location is to view it in a genome browser. However, in the process of getting our answer, we have lost information about the location of these exons on the chromosome. But fear not, Galaxy saves all of your data, so we can recover this information quite easily.
 
 :pencil2: ***Hands on!***
 
-1. First step
-2. Second step
-3. Third step
+1. Open the tool `Compare two Datasets - to find common or distinct rows`
+
+2. Set the parameters to compare the column 4 of the exon file with column 1 of the top-5 exons file to find matching rows.
+
+ ![](../images/101_18.png)
+
+3. Click **Execute** and examine your output file. It should contain the locations of your top 5 exons:
+
+ ![](../images/101_19.png)
+
+4. A good way to learn about these exons is to look at their genomic surrounding. This can be done by using genome browsers. First, we have to tell Galaxy which **Genome build** this data uses (hg19), we can do this as follows:
```

Comment:
Maybe the first sentence can be moved out of this list and a new hands on started. What do you think?